examples/systemcalls/systemcalls.c

```diff
@@ -92,6 +139,48 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
  *   The rest of the behaviour is same as do_exec()
  *   
 */
+	int fd = open(outputfile, O_WRONLY|O_TRUNC|O_CREAT,0644);
```

Comment:
@Jahnavi-pinnamaneni   think about how you could combine logic with the function above.